app/views/shared/forms/_shared_ownership_form.html.erb

```diff
@@ -7,17 +7,22 @@
   <% group_error_class = error ? 'govuk-form-group--error' : '' %>
```

Comment:
The group error class is now being handled by the `govuk_form_group` helper method - so this line can be removed as `group_error_class` is now redundant.